torch/ao/quantization/fx/prepare.py

```diff
@@ -1343,7 +1343,12 @@ def prepare(
     if is_qat:
         additional_qat_module_mapping = prepare_custom_config_dict.get(
             "additional_qat_module_mapping", {})
-        qat_swap_modules(model, additional_qat_module_mapping)
+        if backend_config_dict is None:
```

Comment:
do we plan to leave this checked in for awhile (weeks/months), or will this be gone soon and backend dict be the only path?

Reply:
this will stay here for a while (a few months) until we fully migrate the convert part of fbgemm/qnnpack to use reference models.

hopefully there is not many people using `additional_qat_module_mapping` api, since we'd like to remove it later. I remember we haven't really talked about this in our tutorials, so it's probably OK

Reply:
makes sense, maybe we can add a comment to the code with this context?